lib/git/git_is_repo.fish

```diff
@@ -1,3 +1,7 @@
 function git_is_repo -d "Check if directory is a repository"
-  test -d .git; or command git rev-parse --git-dir >/dev/null 2> /dev/null
+  test -d .git
+  or begin
+    set -l info (git rev-parse --git-dir --is-bare-repository 2>/dev/null)
```

Comment:
this should still be `command git rev-parse`. it's pretty common to override `git` to provide additional functionality or default options, so let's be defensive about this.